server/build.gradle

```diff
@@ -24,11 +24,19 @@ dependencies {
 distributions {
     main {
         contents {
-	    into('client') {
-		from { project(':client').build.outputs.files }
-		fileMode = 0755
-	    }
-	}
+            // Put the client artifacts (HTML, CSS, and JS generated
+            // by Angular) in the `lib/client/public` directory. This
```

Comment:
I'm pretty sure that WebPack is responsible for converting all of the TS into JS. Angular2 is written in TS and I don't believe that it includes a built-in tool for converting to JS (unless we count the JS library which does this conversion *on* the client at "runtime", so to speak...)